pkg/kubernetes/apis/loadtest/v1/spec.go

```diff
@@ -27,9 +33,16 @@ func BuildLoadTestSpec(loadTestType LoadTestType, overwrite bool, distributedPod
 		return lt, ErrRequireTestFile
 	}
 
+	for _, value := range tags {
+		if len(value) > maxTagLength {
```

Comment:
Do we still need this check? Since a type of `LoadTestTags` should be created by [`parseTags`](https://github.com/hellofresh/kangal/pull/39/files#diff-b4c49c47e87f6d9c9e5d6f1275002b2dR223).

Reply:
imo, we should keep the validation because it's k8s thingy but I can remove it if you think it's too much.